test/remote/gateways/remote_litle_test.rb

```diff
@@ -143,9 +143,15 @@ def test_successful_authorization_with_echeck
     assert_equal 'Approved', response.message
   end
 
-  def test_avs_and_cvv_result
+  def test_avs_result
```

Comment:
💬  Comment:

Now that you are spliting the avs and cvv tests, maybe is a good idea to improve the semantics of the test name definition, "test_avs_result" doesn't tell you what are you testing.

Reply:
Thanks for the feedback, corrected! 👍